Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1036,3 +1036,9 @@
 #if !defined(ENABLE_QUICKLOOK_SANDBOX_RESTRICTIONS) && PLATFORM(IOS)
 #define ENABLE_QUICKLOOK_SANDBOX_RESTRICTIONS 1
 #endif
+
+#if !defined(ENABLE_NOTIFY_BLOCKING) \
+    && (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 150000) \
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED > 170400)
```

Comment:
What about WatchOS, TVOS?

Reply:
That is a very good point. Will add!

Thanks for reviewing!

Reply:
Actually, we may want to enable these at a later point here.